assets/src/scss/style.scss

```diff
@@ -50,6 +50,7 @@ Text Domain: planet4-master-theme
 @import "layout/skewed-overlay";
 @import "layout/tables";
 @import "layout/featured-action";
+@import "layout/wide-blocks";
```

Comment:
Importing this here means that it will always be loaded, right? Shouldn't it be loaded only when the "Planet 4 all blocks" feature is enabled, to avoid duplications with the plugin styles? 🤔 Maybe then we should import it in `blocks.scss` instead for example